src/lightning/fabric/strategies/fsdp.py

```diff
@@ -292,6 +293,10 @@ def setup_module(self, module: Module) -> Module:
                 **self._fsdp_kwargs,
             )
 
+        # FSDP doesn't move modules without parameters (e.g. Metrics) to the device
+        # https://github.com/pytorch/pytorch/issues/113113
```

Comment:
If you put this comment inside the function, you don't have to write it in both fabric and pytorch

Reply:
good idea!